src/plugins/discover/public/application/components/context_app/context_app_legacy.tsx

```diff
@@ -111,18 +115,24 @@ export function ContextAppLegacy(renderProps: ContextAppProps) {
 
   return (
     <I18nProvider>
-      <React.Fragment>
-        <ActionBar {...actionBarProps(PREDECESSOR_TYPE)} />
-        {loadingFeedback()}
-        {isLoaded ? (
-          <EuiPanel paddingSize="none">
-            <div className="discover-table">
-              <DocTableLegacy {...docTableProps()} />
-            </div>
-          </EuiPanel>
-        ) : null}
-        <ActionBar {...actionBarProps(SUCCESSOR_TYPE)} />
-      </React.Fragment>
+      {isFailed ? (
+        <ContextErrorMessage status={status} reason={renderProps.reason} />
+      ) : (
+        <EuiPage>
+          <EuiPageContent paddingSize="s" className="cxtAppContent">
```

Comment:
@majagrubic I think this should have the `.dsc` prefix?
```suggestion
          <EuiPageContent paddingSize="s" className="dscCxtAppContent">
```